libmscore/style.h

```diff
@@ -1101,17 +1187,31 @@ class MStyle {
       std::array<QVariant, int(Sid::STYLES)> _values;
       std::array<qreal, int(Sid::STYLES)> _precomputedValues;
 
+      QPageSize   _pageSize;
+      MPageLayout _pageOdd;
+      MPageLayout _pageEven;
+      bool _isMMInch;       ///!!!for 3.01 to 3.01+ conversion
+      QMarginsF _marginsOdd; ///!!!Travis workaround
```

Comment:
These are part of a more refined workaround that did not work on Travis.  I will remove them from here and style.cpp, where they are fully commented out.